pkg/cli/cmd/bicep/publish/publish.go

```diff
@@ -237,7 +237,7 @@ func (r *Runner) prepareSource(ctx context.Context) (*memory.Store, error) {
 	}
 
 	// Push config blob
-	configDesc, err := pushBlob(ctx, configMediaType, nil, src)
+	configDesc, err := pushBlob(ctx, configMediaType, []byte("{}"), src)
```

Comment:
Can you add a comment here? I think it would be helpful to link the other issue where we found the root cause.

Reply:
Added a comment , also linked the issue for reference

Reply:
Looks great! thanks.